scripts/build-daemon.sh

```diff
@@ -0,0 +1,25 @@
+set -e
+set -o pipefail
+
+STAGE=$1
+
+if [ -z $STAGE ]; then
+    echo "This scripts expects the stage as a parameter";
+    exit 1;
+fi
+
+if [ -z "$ACCOUNT_ID" ]; then
+    echo "Please export a ACCOUNT_ID env var before running this";
+    exit 1;
+fi
+
+commit=`git rev-parse HEAD`;
+timestamp=`date +%s`;
+export DOCKER_IMAGE_TAG="$1-$commit-$timestamp";
```

Comment:
Maybe you should use `$STAGE` here, instead of `$1` directly, for readability.

Reply:
Fixed, thanks